AAI/AAIConnectProfile.md

```diff
@@ -98,6 +112,15 @@ described herein, encoded via JWS Compact Serialization per
 [RFC7515](https://datatracker.ietf.org/doc/html/rfc7515), containing
 `ga4gh_passport_v1` as a space-separated entry within the `scope` claim but
 does not contain [GA4GH Claims](#term-ga4gh-claim).
+<a name="term-embedded-token-issuer"></a> **Embedded Token Issuer** --
+a service that signs [Embedded Tokens](#term-embedded-token). This service
+may be:  
+* a [Broker](#term-broker) itself
+* a Broker may use this service as part of collecting GA4GH Claims that the Broker includes in responses from its /userinfo or /token endpoint.
+
+<a name="term-token-container-issuer"></a> **Token Container Issuer** --
```

Comment:
From Security Subgroup call: "Token Container" right now is just a "Passport Token". "Token Container Issuer" could be a "Passport Issuer".

Reply:
Updated